apps/sms/js/dialog.js

```diff
@@ -21,6 +21,7 @@
     id: 'localizableStringWithArgument',
     args: { n: count }
   },
+  classes: ['specific-class'], // optional
```

Comment:
ya the original options seems only for button(or we don't utilize it for other purpose before). I have no preferance about this so let's keep it.